cmake/libpython/CMakeLists.txt

```diff
@@ -532,7 +532,7 @@ if(BUILD_SHARED)
         target_link_libraries(${targetname} ${python3stub_lib})
     endif()
 
-    if(IS_PY3 AND UNIX AND NOT APPLE)
+    if(IS_PY3 AND (LINUX OR BSD))
```

Comment:
For now, I will skip this last commit, it would prevent the flag from being added on unix system other that Linux.

A better solution could be to add test similar to what was done in https://github.com/commontk/CTK/commit/52551d4eb32c53eed34142b2cf21c80331ace718

I will enter a dedicated issue to track this